locales/it-IT.json

```diff
@@ -134,8 +134,8 @@
       "phrases": {
         "0": "Continua.",
         "1": "Continua su <STREET_NAMES>.",
-        "2": "Continue at <JUNCTION_NAME>.",
-        "3": "Continue toward <TOWARD_SIGN>."
+        "2": "Prosegui su <JUNCTION_NAME>.",
```

Comment:
this and the other ones dont match previous translations, the lack of consistency could feel pretty random to a user, imho.

Reply:
I have this weird feeling that they've simply copy-pasted the english instructions into an online translation service, and put the results back here! :raised_eyebrow: